src/popup/App.vue

```diff
@@ -0,0 +1,94 @@
+<template>
+  <div v-bind:class="['popup', {'popup--dark': dark}]">
+    <h2 class="popup__header">Mintbean Editor 2020-04-04 Bryce McMath</h2>
+    <select class="popup__select" name="langs" id="langs">
+      <option v-for="(opt, i) in langOpts" :key="i" :value="opt">{{opt}}</option>
+    </select>
+
+    <editor
+      class="popup__editor"
+      v-model="content"
+      @init="editorInit"
+      :lang="lang"
+      :theme="dark ? 'twilight': 'chrome' "
+      width="500"
+      height="100"
+    ></editor>
+  </div>
+</template>
+
+<script>
+export default {
+  name: 'Popup',
+  components: {
+    editor: require('vue2-ace-editor'),
+  },
+  data() {
+    return {
+      content: '',
+      langOpts: ['css'],
+      lang: 'css',
+    };
+  },
+  computed: {
+    dark() {
+      return window.matchMedia && window.matchMedia('(prefers-color-scheme: dark)').matches;
+    },
+  },
+  methods: {
+    editorInit() {
+      require('brace/ext/language_tools'); //language extension prerequsite...
+      require('brace/mode/html');
+      require('brace/mode/css');
+      require('brace/mode/javascript'); //language
+      require('brace/mode/less');
+      require('brace/theme/chrome');
+      require('brace/theme/twilight');
+      require('brace/snippets/javascript'); //snippet
+    },
+  },
+  watch: {
+    content: function(val) {
+      let tab;
+      chrome.tabs.query({ currentWindow: true, active: true }, tabs => {
+        chrome.tabs.insertCSS(tabs[0].id, { code: val });
```

Comment:
Will this always only work on the first browser tab (`tabs[0]`)?

Reply:
Yes, that was the intent. I figured somebody using this would only want to apply to it one tab at a time

Reply:
Added a comment to make it clear